app/assets/stylesheets/trips.css.scss

```diff
@@ -51,7 +51,11 @@
   position: relative;
   border: solid 3px #363636;
   border-radius: 5px 5px 5px 5px;
-  margin: 15px 15px 15px 15px
+  margin: 15px 15px 15px 15px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `15px` instead of `15px 15px 15px 15px`